packages/service/src/service.ts

```diff
@@ -56,6 +64,17 @@ app.use(passport.session());
 
 app.post("/login", passport.authenticate("local", { successRedirect: "/" }));
 
+app.post("/users", async function (req, resp) {
+  const dbUserHashedPassword = await updatePassword(req.user);
```

Comment:
these urls are getting a little confusing about what they do.

maybe a `put` on `/users/update-password` would make more sense for updating the password?
maybe a `post` on `/users/new` would create a new user?

Reply:
Yes, these are good points